src/authorization-control.ts

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+import type {
+  Marqeta, MarqetaError,
+  MarqetaOptions,
+} from './'
```

Comment:
Why have two on one line, and one on another? We usually have all on one line - if they fit, or one to a line if they don't... Right?